soroban-sdk/src/tests/contract_invoke.rs

```diff
@@ -21,9 +20,15 @@ fn test_invoke() {
 }
```

Comment:
@graydon Could we make this test pass by retaining the panic string and plopping it into a debug event?

There are plenty of examples that rely on the panic string in the test. I would like us to move away from that type of testing and instead use errors, but I don't want to take away the ability to should on the error string if we don't have to.

Reply:
Env tweaked! Posting update.